dosagelib/plugins/webtoons.py

```diff
@@ -89,6 +89,7 @@ def getmodules(cls):
             cls('Blessed', 'drama/blessed', 1193),
             cls('BloodInk', 'action/blood-ink', 1490),
             cls('BloodlessWars', 'sf/bloodless-wars', 1622),
+            cls('BloopBloopRelationshipComic', 'slice-of-life/comedy', 239970),
```

Comment:
Does this work for you? The second parameter needs to be the path to the comic, otherwise you just get a 404 error. Like this:
```suggestion
            cls('BloopBloopRelationshipComic', 'challenge/bloop-bloop-relationship-comic', 239970),
```